web/templates/user/show.html.eex

```diff
@@ -19,5 +19,5 @@
 
 </ul>
 
-<%= link "Edit", to: user_path(@conn, :edit, @user) %>
+<%= link "Edit", to: user_path(@conn, :edit, @user) %>&nbsp;
```

Comment:
It was mostly an example of how to add class to the buttons. I don't care either way, but the buttons in bootstrap deal with spacing pretty well on their own.